projects/components/src/lib/directives/alternative-text/alternative-text.directive.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { AfterViewInit, Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+
+const ALTERNATIVE_TEXT = 'alt';
+
+/**
```

Comment:
Why would this be for clr-icon only?

And what's the value of this besides just adding the alt attribute?

Reply:
Well, my thoughts for the directive were: because I need to set it to the inner svg which is injected inside the clr-icon and that I need to do the same in more than one place.

Reply:
I see, I didn't realize we're adding the tag to something we don't render ourselves.